lib/Doctrine/Migrations/Configuration/ConfigurationLoader.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Configuration;
+
+use Doctrine\Migrations\Configuration\Exception\UnknownLoader;
+use Doctrine\Migrations\Configuration\Loader\ArrayLoader;
+use Doctrine\Migrations\Configuration\Loader\JsonFileLoader;
+use Doctrine\Migrations\Configuration\Loader\Loader;
+use Doctrine\Migrations\Configuration\Loader\PHPFileLoader;
+use Doctrine\Migrations\Configuration\Loader\XmlFileLoader;
+use Doctrine\Migrations\Configuration\Loader\YamlFileLoader;
+use function count;
+
+class ConfigurationLoader
```

Comment:
Can this class be `final`?

Reply:
marking it as final will make `ConfigurationHelperTest` quite more complex. But thanks to your review i noticed that i forgot to mark it as internal. Thanks